fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java

```diff
@@ -54,7 +67,7 @@ public static FHIRRegistry getInstance() {
      *     true if a resource for the given canonical url and resource type exists in the registry, false otherwise
      */
     public boolean hasResource(String url, Class<? extends Resource> resourceType) {
```

Comment:
Add documentation that says you can only call this for definitional resource types and that otherwise it will throw an IllegalArgumentException

Reply:
An `IllegalArgumentException` will **not** be thrown if the resource isn't definitional for this method. The method will simple return false.